azavea/raster-foundry
# Create a bare bones worker.

Create and provision a worker machine with vagrant/packer.
The new machine type is a stripped down version of the app machine. It is
provisioned with the same python dependencies needed for the app servers but
it does not have Nginx and not ports are exposed to the outside world.
Eventually we will create python code that will be run as a service that will
poll SQS and perform tasks (add things back the SQS queue) as necessary.

Connects #115 

To test:
 * Vagrant up should create a new worker vm.
 * Ensure you can access aws credentials at `/home/vagrant/.aws/` and `/var/lib/rf`
